frontend/packages/core/src/AppProvider/index.tsx

```diff
@@ -120,9 +122,22 @@ const ClutchApp: React.FC<ClutchAppProps> = ({
     [workflowSessionStore]
   );
 
-  const appContextValue = React.useMemo(() => ({ workflows: discoverableWorkflows }), [
-    discoverableWorkflows,
-  ]);
+  const appContextValue = React.useMemo(
+    () => ({
+      workflows: discoverableWorkflows,
+      triggerHeaderItem: (item: HeaderItems, open: boolean, data: unknown) =>
```

Comment:
Would it make sense to have this solely be for opening header items? My thought here is that if a header item is already open then there is probably no point in having this be used to close it since a click away would probably do that anyways. It would also simplify this logic since we would only need to support triggering an open.

Reply:
Yeah I was going back and forth on this one, trying to think of an instance where we might want to close it outside of the header. I'll remove the open check and simplify the logic and if we run into an instance where it's necessary we can always extend it

Reply:
Removed the open boolean and am using whether the data is present or not to signify it's been triggered (which it was already doing anyways).